### PR TITLE
fix: usage of `a deprecated Node.js version`

### LINKED ISSUE #21947: `dashboards-test.yml` uses `a deprecated Node.js version`
https://github.com/highcharts/highcharts/actions/runs/11119563751

### PR DESCRIPTION
**Pull request by hamirmahal**
fixes #21947.

### CODE REVIEW

File: .github/workflows/dashboards-test.yml [hunk 1]
```diff
@@ -34,13 +34,13 @@ jobs:
         node-version: [lts/*]
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```
Comment (hamirmahal) [inline .github/workflows/dashboards-test.yml:37]: This is the latest major version at the time of this writing.

**Comment by hamirmahal** [comment]
@TorsteinHonsi what are your thoughts on this pull request?

**Comment by TorsteinHonsi** [comment]
@hamirmahal Thanks for the PR! @goransle can you take a look?

**Comment by goransle** [comment]
The fix looks good. I would prefer to keep the single-quotes to stay consistent with the other workflows though,

**Comment by hamirmahal** [comment]
@TorsteinHonsi You're welcome! Feel free to let me know if I should make any changes.

**Review by goransle — APPROVED** [review]
Thanks, looks good 👍

**Comment by hamirmahal** [comment]
You're welcome!

**Pull request merged**